task/clamav-scan/0.1/clamav-scan.yaml

```diff
@@ -123,18 +125,23 @@ spec:
         #!/usr/bin/env bash
         source /utils.sh
 
-        HACBS_ERROR_OUTPUT=$(make_result_json -r "ERROR")
         if [ -f /tekton/home/clamscan-result.json ];
         then
           cat /tekton/home/clamscan-result.json
           INFECTED_FILES=$(jq -r '.infected_files' /tekton/home/clamscan-result.json || true )
           if [ -z "${INFECTED_FILES}" ]; then
             echo "Failed to get number of infected files"
+            note="Task $(context.task.name) meets error: Failed to get number of infected files from /tekton/home/clamscan-result.json, please check the tekton task log"
```

Comment:
```suggestion
            note="Task $(context.task.name) failed: Unable to get number of infected files from /tekton/home/clamscan-result.json, please check the tekton task log"
```

Reply:
what about 

note="Task $(context.task.name) failed: Unable to get number of infected files from `/tekton/home/clamscan-result.json`. Check the Tekton task log." ?

a few things Red Hat requires of error messages: 

- They have to be completely clear about where a user can find help. Will they know right away where to find the log?

- They have to tell users how to make corrections. Will the log do that?

Reply:
Updated the note to `Task $(context.task.name) failed: Unable to get number of infected files from `/tekton/home/clamscan-result.json`. For details, check the Tekton task log.`

Reply:
Thanks for the questions. They make sense.